components/places/ffi/src/lib.rs

```diff
@@ -406,6 +406,22 @@ pub extern "C" fn places_get_visit_page(
     })
 }
 
+#[no_mangle]
+pub extern "C" fn places_accept_result(
+    handle: u64,
+    search_string: FfiStr<'_>,
+    url: FfiStr<'_>,
+    error: &mut ExternError,
+) {
+    log::debug!("places_accept_result");
+    CONNECTIONS.call_with_result(error, handle, |conn| -> places::Result<_> {
+        let search_string = search_string.as_str();
+        let url = parse_url(url.as_str())?;
```

Comment:
As I mentioned, we probably want to just log a warning in the case that the url is invalid, rather than throwing. Something like this maybe:

```suggestion
        let url = if let Ok(url) = parse_url(url.as_str()) {
            url
        } else {
            log::warn!("Ignoring invalid URL in places_accept_result");
            return Ok(());
        };
```

Reply:
Really sorry, I totally forgot this one. Done.

Reply:
No need to be sorry, that's what the review is for.